src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -280,6 +318,18 @@ public function getPath()
      */
     final public function normalize($value)
     {
+        if (is_string($value) && isset(self::$placeholders[$value])) {
+            foreach (self::$placeholders[$value] as $placeholder) {
+                try {
+                    $this->normalize($placeholder);
+                    break;
```

Comment:
should be `return $value;`
and the current return below should be `throw $e;`

Reply:
now with test :)